app/controllers/signatures_controller.rb

```diff
@@ -32,7 +32,7 @@ def verify
     @petition = @signature.petition
 
     if @signature.validated?
-      flash[:notice] = "Thank you. Your signature has already been added to the <span class='nowrap'>e-petition</span>."
+      flash[:notice] = "Thank you. Your signature has already been added to the <span class='nowrap'>petition</span>."
```

Comment:
We don't need the class any more as the word isn't hyphenated.... So it won't wrap.